DEVELOPER.md

```diff
@@ -49,286 +13,370 @@ To test and run services, you need to get access to the development Kubernetes c
 Our guide uses Minikube as an example because it is easy to get it started, but you can use another Kubernetes cluster as well.
 Please see [Kubernetes](https://kubernetes.io) for setting up a development cluster.
 
-Please read [doc](https://www.fluvio.io) for a technical arch and operation guide. 
+Please read [doc](https://www.fluvio.io) for a technical arch and operation guide.
 
 ---
 
 ## Setting up Development Environment
 
 The development environment requires the Rust toolchain and a few other build dependencies to build changes to the source code.
 
-To deploy and test the source code, Kubernetes is required because Fluvio stores its metadata in Kubernetes. Fluvio binaries can be tested locally, running outside of any k8s cluster using the `--local` parameter or they can be built into a docker image and deployed into the Kubernetes cluster.
+Fluvio allows you to run the cluster locally or in a Kubernetes cluster, when
```

Comment:
This doesn't seems to be need in this section.  Move K8 related to k8 section.  Only add requirement for local cluster

Reply:
Okay I removed this to make simpler to read, users will have to reach the Kubernetes section to learn more.